docs/.vuepress/public/presets/toml/gruvbox-rainbow.toml

```
@@ -0,0 +1,148 @@
+"$schema" = 'https://starship.rs/config-schema.json'
+
+format = """
+[](#d65d0e)\
+$os\
+$username\
+[](bg:#d79921 fg:#d65d0e)\
+$directory\
+[](fg:#d79921 bg:#689d6a)\
+$git_branch\
+$git_status\
+[](fg:#689d6a bg:#458588)\
+$c\
+$rust\
+$golang\
+$nodejs\
+$php\
+$java\
+$kotlin\
+$haskell\
+$python\
+[](fg:#458588 bg:#665c54)\
+$docker_context\
+[](fg:#665c54 bg:#3c3836)\
+$time\
+[ ](fg:#3c3836)\
+$line_break$character"""
+
+[os]
+disabled = false
+style = "bg:#d65d0e fg:#fbf1c7"
+
+[os.symbols]
+Windows = "󰍲"
+Ubuntu = "󰕈"
+SUSE = ""
+Raspbian = "󰐿"
+Mint = "󰣭"
+Macos = "󰀵"
+Manjaro = ""
+Linux = "󰌽"
+Gentoo = "󰣨"
+Fedora = "󰣛"
+Alpine = ""
+Amazon = ""
+Android = ""
+Arch = "󰣇"
+Artix = "󰣇"
+CentOS = ""
+Debian = "󰣚"
+Redhat = "󱄛"
+RedHatEnterprise = "󱄛"
+
+[username]
+show_always = true
+style_user = "bg:#d65d0e fg:#fbf1c7"
+style_root = "bg:#d65d0e fg:#fbf1c7"
+format = '[ $user ]($style)'
+
+[directory]
+style = "fg:#fbf1c7 bg:#d79921"
+format = "[ $path ]($style)"
+truncation_length = 3
+truncation_symbol = "…/"
+
+[directory.substitutions]
+"Documents" = "󰈙 "
+"Downloads" = " "
+"Music" = "󰝚 "
+"Pictures" = " "
+"Developer" = "󰲋 "
+
+[git_branch]
+symbol = ""
+style = "bg:#689d6a"
+format = '[[ $symbol $branch ](fg:#fbf1c7 bg:#689d6a)]($style)'
+
+[git_status]
+style = "bg:#689d6a"
+format = '[[($all_status$ahead_behind )](fg:#fbf1c7 bg:#689d6a)]($style)'
+
+[nodejs]
+symbol = ""
+style = "bg:#458588"
+format = '[[ $symbol ($version) ](fg:#fbf1c7 bg:#458588)]($style)'
```

One of the spaces should go along `$version` into the `()` block to avoid printing two consecutive spaces if `$version` is empty.

```suggestion
format = '[[ $symbol( $version) ](fg:#fbf1c7 bg:#458588)]($style)'
```